app/services/service_provider_seeder.rb

```diff
@@ -80,6 +80,12 @@ def check_for_missing_sps
     extra_sp_error = ExtraServiceProviderError.new(
       "Extra service providers found in DB: #{extra_sps.join(', ')}",
     )
-    NewRelic::Agent.notice_error(extra_sp_error)
+
+    if IdentityConfig.store.team_ursula_email.present?
+      ReportMailer.warn_error(
+        email: IdentityConfig.store.team_ursula_email,
+        error: extra_sp_error,
+      ).deliver_later
```

Comment:
Updating the infra is low complexity but slightly longer time (since we have to wait for the 1x/week infra deploy for the changes). There's not a huge rush to this so it's probably fine

Reply:
Made a PR to update the infra here: https://github.com/18F/identity-devops/pull/6316